runbotics/runbotics-orchestrator-ui/src/src-app/components/cron/types.ts

```diff
@@ -313,3 +341,25 @@ export interface Clicks {
     time: number;
     value: number;
 }
+
+export interface cronObjProps {
```

Comment:
always start interfaces with capital letter

Reply:
missed it, done